articles/getting-started-with-quasar-framework/index.md

```diff
@@ -0,0 +1,295 @@
+layout: engineering-education
+status: 
+published: 
+url: /engineering-education/getting-started-with-quasar-framework/
+title: Getting Started with Quasar Framework
+description: In this article, we will look at get started with quasar framework, its components structure, its pros and finally cons.
+author: espira-marvin
+date: 
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/getting-started-with-quasar-framework/hero.jpg
+    alt: Getting Started with Quasar
+---
+
+### Introduction
+
+Quasar Framework is a Vue.js based framework used to develop cross-platform applications using one codebase saving development costs. Quasar offers a state of art user Interface and support builds for:
+  - single-page applications 
+  - progressive web applications
+  - server-side rendering
+  - mobile apps (Ios and android) using cordova or capacitor 
+  - multi-platform desktop apps using electron 
+  - browser extensions. 
+  
+The overhead of using quasar is minimal as it implements the best practices in web development out-of-the-box, taking into account performance and responsiveness in mind. It also has components for almost every web development need.
+
+### Prerequisites
+
+1. Knowledge of Vue js
+2. Quasar CLI
+3. Node.js 10.x or newer excluding 13 and 15
+4. Npm 5.10 or newer / yarn 1.2 or newer
+
```

Comment:
In the prerequisites section, you outline the things the reader needs prior to going through your tutorial. The reader does not need Quasar CLI at this point. This is why you are guiding the installation deeper in your tutorial.
```suggestion
To follow along with this tutorial, you need:
1. Some knowledge of Vue js
2. Node.js 10.x or newer excluding 13 and 15.
3. Npm 5.10 or newer / yarn 1.2 or newer.

```